File: engine/engine-model/src/main/scala/tools/packet/spawn/StopMonsterControl.scala
package tools.packet.spawn

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class StopMonsterControl(private var _monsterId: Int) extends PacketInput {
  def monsterId: Int = _monsterId

  override def opcode(): SendOpcode = SendOpcode.SPAWN_MONSTER_CONTROL
}

File: engine/engine-model/src/main/java/tools/packet/spawn/StopMonsterControl.java
package tools.packet.spawn;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record StopMonsterControl(Integer monsterId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SPAWN_MONSTER_CONTROL;
   }
}